airflow/bin/cli.py

```diff
@@ -1339,6 +1340,9 @@ class CLIFactory(object):
         'dag_regex': Arg(
             ("-dx", "--dag_regex"),
             "Search dag_id as regex instead of exact string", "store_true"),
+        'no_dag_reset': Arg(
+            ("--no_dag_reset", ),
+            "Do not reset DAG state after clearing tasks", "store_true"),
```

Comment:
Rather than having to do `reset_dag_runs=not args.no_dag_reset` at the call site, would we do something like `"store_false", metavar="reset_dag_runs", default=True)`?

Reply:
metavar was not what I meant.

`'reset_dag_runs': Arg(... "store_false", default=True)` was what I meant.

Reply:
Not possible without further changes; the `Arg()` namedtuple doesn't include `dest`, and the generated variable name is not set by the dictionary key; you'd have to do `--reset_dag_runs=0` or something similar, which seems unnecessarily ugly.